rustls/src/common_state.rs

```diff
@@ -186,6 +187,78 @@ impl CommonState {
         self.send_plain(data, Limit::Yes)
     }
 
+    pub(crate) fn eager_send_some_plaintext(
```

Comment:
Could this fn be updated with some comments describing the process being undertaken? It's a little tricky to follow.

Reply:
I think this feedback is still unaddressed.